xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEStream.cpp

```diff
@@ -180,9 +178,7 @@ void CActiveAEStream::RemapBuffer()
     }
 
     // swap sound packets
-    CSoundPacket *tmp = m_currentBuffer->pkt;
-    m_currentBuffer->pkt = m_remapBuffer;
-    m_remapBuffer = tmp;
+    std::swap(m_currentBuffer->pkt, m_remapBuffer);
```

Comment:
Looks like this is defined in `<string_view>` now?

Reply:
I don't think so?

https://en.cppreference.com/w/cpp/memory/unique_ptr/swap2